### DISCUSSION TITLE
Add recurrence support

### DISCUSSION THREAD

**Discussion by GitToTheHub** [excid3/simple_calendar#328] [Ideas]
Hi,

it would be awesome, if you could add recurrence support. You could expect a recurrence_ice_cube_rule-column in the model. You read this Rule, and know when to repeat the event. This would make the recurrence-implementation so much easier, because othwerwise you have to make your own solution, which is not "simple".

Regards,

Manuel

**Comment by excid3** [comment]
Simple Calendar does not provide models so you can use any recurring events library you want with it. All you have to do is provide the events.

**Comment by GitToTheHub** [comment]
Sorry, maybe I expressed myself wrong.

I mean you could expect a new attribute like start_time and end_time. This new attribute could be a recurrence-Attribute. The user could deliver its recurrence-rule for the event and simple_calendar would deliver the event in the do block for the calendar. So the user have not to implement its own implementation like making a new Array with all recurring events by for e.g. dubbling the active record entries.
